site/pages/rest-api/examples/create-analysis.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Create Analysis
+---
+
+Create a new Analysis for a site.
+
+# cURL
+
+```bash
+curl -X POST --data-binary @analysis.xml --user {username}:{password} https://{server}:{port}/api/analyses/{site}
+```
+
+# python
+
+```python
+import requests
+with open('analysis.xml', 'rb') as f:
```

Comment:
I think `f` should be changed to `xml` for now, for consistency with other files